README.md

```diff
@@ -1047,10 +1047,13 @@ argument to specify your new model:
 
 If you are happy with your newly-trained model and wish to upload it to a supported endpoint, you can do so with the `ilab model upload` command.
 
-InstructLab currently supports uploading to [Hugging Face](https://huggingface.co/) and OCI registires such as [Quay.io](https://quay.io/), with plans to support S3 uploads as well.
+InstructLab currently supports uploading to [Hugging Face](https://huggingface.co/), OCI registries such as [Quay.io](https://quay.io/), and [AWS S3](https://aws.amazon.com/s3/) buckets.
```

Comment:
Can you add examples later on in the README showing how to upload to S3?

I only mention this because S3 URLs commonly take on one of two different syntaxes:

 1. Path style: `s3.amazonaws.com/<bucket>/<key>`
 2. Virtual hosted style: `<bucket>.s3.amazonaws.com/<key>`

I've also used global s3 URLs (basically ones that start with `s3://` instead of `http://`) and access points.

If we don't want `s3://` being used, then we should definitely prohibit it.

Reply:
Yes, added some examples!